snap/snapcraft.yaml

```diff
@@ -63,6 +63,41 @@ parts:
       - libssh-4
       - libwind0-heimdal
 
+  sdk:
```

Comment:
Don't think we should be including deb file within the snap. Since the recommendation is for DU to include the libDO.so in its own snap, let's keep this snapcraft file for the DO agent alone.

Reply:
Agreed. This is for testing purposes only.